### PR TITLE
Test2

### PR DESCRIPTION
**Pull request by SamitIntern**
Please let me know your thoughts about this PR.

### CODE REVIEW

**Review by shreyasgune — APPROVED** [review]
looks good. Wait on Azraf's comments and then you can merge.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+***
+
+Installing and Starting the Application
+----------------------------------------
+
+To run the application, follow the following instructions: -.
+
+After the repository is cloned in your local machine, make sure you are present in the same directory where the repository is cloned.
+
+Type in the command '"docker build --tag=<Any tag name of your choice>"'.
```
Comment (AzrafMedic) [inline README.md]: You should format code using ``` code example ``` so its easier to read.

Comment (AzrafMedic) [reply README.md]: please read the document i sent you on commenting and markdown practices

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+To make sure that the command executed properly, type '"docker image ls"'.
```
Comment (AzrafMedic) [inline README.md]: Format code properly within markdown

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+You should see a *docker image* with the name of the tag that you gave in the previous command.
+
+Now to run the application, type '"docker run --p 8080:<A port of your choice> <The tag name you mentioned previously>"'.
```
Comment (AzrafMedic) [inline README.md]: Format code properly within markdown

**Review by AzrafMedic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SamitIntern) [reply README.md]: I thought this is a command, therefore I didn't use the code format. Should I use the code format for commands as well going forward ?

Comment (SamitIntern) [reply README.md]: And yes, I did read the document

Comment (AzrafMedic) [reply README.md]: yes you should use the code format for commands so it can help engineers differentiate code from instructions

**Review by AzrafMedic — APPROVED** [review]
Good work

**Pull request merged**